crates/server/src/authority/catalog.rs

```diff
@@ -365,20 +368,18 @@ impl Catalog {
             // if this is empty then the there are no authorities registered that can handle the request
             let response = MessageResponseBuilder::new(Some(request.raw_query()));
 
-            let result = send_response(
+            send_response(
                 response_edns,
                 response.error_msg(request.header(), ResponseCode::Refused),
                 response_handle,
             )
-            .await;
+            .await
+            .map_err(|e| {
+                debug!("failed to send response: {}", e);
```

Comment:
There was a patch added by Gytis that reduced the log levels from trust-dns, probably because of spam in logs. Every warn/error log was changed to debug.

Reply:
Is that the commit you are referring to: https://github.com/NordSecurity/trust-dns/commit/fe423f88f82a43d5b0345fe51b779d77dbf72a4f - it's a pity that we will never know *why* the change was made ;)